_data/marwahmanbir/marwahmanbir.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Manbir Singh Marwah
+institution: Guru Gobind Singh Indraprastha University, Delhi
+profile_pic: manbir.jpg
+quote: If you ever think of giving up, remember why you started. You didn't come this far, to only come this far.
```

Comment:
The maximum limit for a quote is 100 characters, you quote contains 106. More details [here](https://github.com/education/graduation#3-add-your-profile-information)